setup.py

```diff
@@ -13,10 +13,10 @@
 PYTYPE = ["pytype==2022.7.26"] if IS_NOT_WINDOWS else []
 if IS_NOT_WINDOWS:
     # TODO(adam): use this for Windows as well once PyPI is at >=1.6.1
-    STABLE_BASELINES3 = "stable-baselines3>=1.6.0"
+    STABLE_BASELINES3 = "stable-baselines3[extra]>=1.6.0"
```

Comment:
FWIW `gym[atari]` just gets a version of `ale-py` which isn't the version we want to use, and `gym[accept-rom-license]` just gets `autorom[accept-rom-license]~=0.4.2`, so we don't actually need to add a dependence to `gym` to that key.

Reply:
I see. I assume it gets 0.7.5 because of the ~=0.7.1 version tag getting the latest 0.7.x one, and we need 0.7.4 for some reason?

Specifying `ale-py` and `autorom` versions directly may be the least-evil option here then.

Hopefully soon we can upgrade to latest Gym once SB3 supports it and the need to pin all these versions will go away.

Reply:
> I see. I assume it gets 0.7.5 because of the ~=0.7.1 version tag getting the latest 0.7.x one, and we need 0.7.4 for some reason?

Yep. From memory: `gym 0.21.0` has this weird thing where when running atari environments it creates a numpy RNG and calls some method on it that isn't actually supported. `ale-py 0.7.4` fixes that problem.